Policies/github-acceptable-use-policies.md

```diff
@@ -50,10 +50,21 @@ While using the Service, under no circumstances will you:
 
 - violate the privacy of any third party, such as by posting another person's personal information without consent.
 
-### 4. Services Usage Limits
+### 4. Spam and Inauthentic Activity on GitHub
+Automated excessive bulk activity and coordinated inauthentic activity, such as spamming, are prohibited on GitHub. Prohibited activities include:
+* bulk distribution of promotions and advertising prohibited by GitHub terms and policies
+* inauthentic interactions, such as fake accounts and automated inauthentic activity
+* rank abuse, such as automated starring or following
+* creation of or participation in secondary markets for the purpose of the proliferation of inauthentic activity
+* using GitHub as a platform for propagating abuse on other platforms
+* phishing or attempted phishing
+
+GitHub reserves the right to remove any Content in violation of this policy.
```

Comment:
in the event github removes content (for the reasons above or any reason for that matter) what is the expectation to the end user? is it just... gone forever? is it potential to recover the data and move it somewhere else? do we get a warning or anything? deprecation?